resume_pj/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!-- Here we import the Boostrap CSS stylesheet -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="./css/mainR.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/> 
+    <title>Domenico DC - Responsive grid project</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+            <div class="container-fluid">
+                <div class="row w-100">
+                    <div class="col-10">
+                        <a class="navbar-brand" href="#">Maya Nelson</a>
+                        <a class="d-block" href="#">PROJECT MANAGER</a>
+                    </div>
+              <button class="navbar-toggler col-2" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+              </button>
+              <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+                <div class="navbar-nav">
+                  <a class="nav-link active" aria-current="page" href="#">Home</a>
+                  <a class="nav-link" href="#">Features</a>
+                  <a class="nav-link" href="#">Pricing</a>
+                  <a class="nav-link disabled" href="#" tabindex="-1" aria-disabled="true">Disabled</a>
+                </div>
+              </div>
+            </div>
+            </div>
+          </nav>
+    </header>
+
+
+    <main>
+        <div class="border border-warning container-fluid">
+            <div class="row border border-primary ">
+                <div class="col-10 offset-1 ">
+                    <div class="row justify-content-center text-center background-beige">
+               
+                    <img src="./img/photo-profile.jpeg" class="rounded-circle col-5" alt="profile pic">
+                    <h2>Maya Nelson</h2>
+                    <hr>
+                   
+                        <div class="row justify-content-around">
+                            
+                                <button type="button" class="col-5 btn btn-outline-primary">RESUME</button>
+                            
+                            
+                                <button type="button" class="col-5 btn btn-outline-primary">PROJECTS</button>
```

Comment:
Don't use Boostrap buttons please. Use Bootstrap just for the layout / responsive behaviour. Your buttons and everything else should be styled by you with custom CSS.